### ISSUE TITLE
Create config files for all models + add how to in README

### ISSUE THREAD

**Issue by TimDettmers** [uclmr/jack#319]
- [x] SNLI: ESIM
- [ ] SNLI: BiLSTM
- [x] SQuAD: BiDAF
- [x] Update README

**Comment by riedelcastro** [comment]
Is this done or obsolete? This is the one issue holding us back from closing `v0.1` :)

**Comment by dirkweissenborn** [comment]
I think this is almost done, we have a dedicated QA page with downloadable models and stats on SQuAD. We would still need the same for NLI though. I think @pminervini made progress on that .

**Comment by pminervini** [comment]
There is some progress on that: https://github.com/uclmr/jack/issues/350

Adding some sort of `SNLI: BiLSTM` conf file right now

BTW the performance of the DAM model is sub-par (40-something % dev accuracy) - I don't know why, because I'm getting 87% dev accuracy when I use it with the very same hyperparameters on another codebase. A cause might be the learning rate decay that kicks in by default.

**Comment by dirkweissenborn** [comment]
ok, let me fix ESIM. Concerning DAM I do not think that it is the learning rate decay that gives these problems. I will look at it myself.
